src/modules/root-container/utils/focus-within.ts

```diff
@@ -31,7 +38,7 @@ const focusWithin = (
     let isFocused: boolean;
 
     const action = () => {
-      const activeElement: any = document.activeElement;
+      const activeElement = canUseActiveElement ? document.activeElement : null;
```

Comment:
Yes, it is working. In general, it will work as worked before - `activeElement` will be `null` when IE will try to get access to the parent of top element. In such case, instead of throwing an error, recursion will just stop.